cryptobox/crypto_box_easy.go

```diff
@@ -105,6 +108,7 @@ func CryptoBoxOpenDetached(c []byte, mac []byte, n []byte, pk []byte, sk []byte)
 }
 
 func CryptoBoxOpenEasyAfterNm(c []byte, n []byte, k []byte) ([]byte, int) {
+	support.CheckSizeMin(c, CryptoBoxMacBytes(), "msg")
```

Comment:
I would use `"ciphertext"` because `c` is the ciphertext, not the message. It would also be great if you could change the `&m[0]` below to `support.BytePointer(m)`.